project-base/tests/ShopBundle/Test/FunctionalTestCase.php

```diff
@@ -21,6 +25,16 @@ abstract class FunctionalTestCase extends WebTestCase
      */
     private $domain;
 
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Router\DomainRouterFactory
+     */
+    protected $domainRouterFactory;
```

Comment:
You have three different visibilities set in this class in this PR, can you please fix it?

Reply:
Fixed https://github.com/shopsys/shopsys/pull/1471/commits/ea6f0fb83090f788cf5a9f7908a67f1157851f32